htdocs/ticket/class/ticket.class.php

```diff
@@ -1454,6 +1463,9 @@ public function markAsRead($user, $notrigger = 0)
 		$error = 0;
 
 		if ($this->statut != self::STATUS_CANCELED) { // no closed
+			if (empty($notrigger)) {
+				$this->oldcopy = dol_clone($this, 1);
```

Comment:
We should not set the   oldcopy inside the update method.
Because inside this method, the property were already set to the new value.
The oldcopy must be set by the caller of update, before setting the new values.